web-extension-1/src/popup/popup.tsx

```diff
@@ -1 +1,22 @@
-console.log('Popup script');
\ No newline at end of file
+import { MessagePopupToContentI } from 'interfaces';
+
+(() => {
+  const sendPopupIsOpened = () => {
+    const gettingActiveTab = browser.tabs.query({ active: true, currentWindow: true });
+
+    gettingActiveTab.then(tabs => {
+      try {
+        const tabId = tabs[0].id;
+        if (tabId) {
+          browser.tabs.executeScript(tabId, { file: 'content-scripts.js' });
```

Comment:
Why is the popup executing the content-script which then communicates with the background page?
The popup can directly communicate with the background page. How would you do that?